lib/review_bot/pull_request.rb

```diff
@@ -2,7 +2,7 @@
 module ReviewBot
   class PullRequest < SimpleDelegator
     def needs_review?
-      !approved? && !blocked? && !review_in_progress?
+      !approved? && !ez_label_approved? && !blocked? && !review_in_progress?
```

Comment:
Rather than add another conditional on this line, what do you think about changing the definition of `approved?`

Reply:
👍